waterdata/templates/monitoring_location.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    {% if status_code == 200 %}
+        <h3>Station Name: {{ station_name }}</h3>
```

Comment:
Semantically we should probably be using h1.  @jkreft-usgs, opinion?

Reply:
I'm also adding a title block to set the page's title as part of WDFN-26. I'm thinking we may want to put the station name in the <title> tag too.

Reply:
definitely should be using h1- and yes, the station name should be in the title block

Reply:
Sure. The only reason h3 is being used is because that's the tag that's being using on the index. I don't really care one way or the other.